packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -16,49 +16,48 @@ import get from 'lodash/get';
 import flattenDeep from 'lodash/flattenDeep';
 import difference from 'lodash/difference';
 import isEqual from 'lodash/isEqual';
-import DetailsPane from '../DetailsPane/DetailsPane';
-import FilterPanel from '../FilterPanel/FilterPanel';
-import MetadataPane from '../MetadataPane/MetadataPane';
-import NavigationToolbar from '../NavigationToolbar/NavigationToolbar';
-import CIDocumentContent from '../CIDocumentContent/CIDocumentContent';
-import processDoc from '../../../../utils/document/processDoc';
-import { getEnrichmentName } from '../../utils/enrichmentUtils';
-import { getDetailsFromItem, getDetailsFromMetadata } from '../../utils/details';
-import { getFilterHelper, ProcessFilter } from '../../utils/filterHelper';
-import { isFilterEmpty, updateFilter } from '../../utils/filterUtils';
-import { getId, findElement, findElementIndex } from '../../../../utils/document/idUtils';
+import DetailsPane from '@CIDocument/components/DetailsPane/DetailsPane';
+import FilterPanel from '@CIDocument/components/FilterPanel/FilterPanel';
+import MetadataPane from '@CIDocument/components/MetadataPane/MetadataPane';
+import NavigationToolbar from '@CIDocument/components/NavigationToolbar/NavigationToolbar';
+import CIDocumentContent from '@CIDocument/components/CIDocumentContent/CIDocumentContent';
+import processDoc from '@rootUtils/document/processDoc';
```

Comment:
could we just do `utils`? also not sure if its for this PR but i wonder if doing something like `import { processDoc } from 'utils/document';` reads easier

Reply:
See my note above on why I went with `rootUtils` (to distinguish from the utils nested in component folders). I agree that updates like that would make imports read easier, and I think it could make sense to do in this PR, so I'll go through and do that :)